### ISSUE TITLE
Pop ups show NaNs and Infinities as null

### ISSUE THREAD

**Issue by dgaubert** [CartoDB/Windshaft-cartodb#705]
Because of having been converted to JSON.

**Comment by dgaubert** [comment]
CR: #706

**Comment by jgoizueta** [comment]
### Acceptance [fail 😞 ]

I created a table with a numeric column containing some infinite/nan values.

As soon as the column is used for styling a map an error occurs (it is returned by the map instantiation). It seems that, when computing the colour ramp the text values such as `inf` are not being interpreted/converted as numeric.

The map instantiation returns a 200 response with this body:

```javascript
/**/ typeof _cdbc_3148608122_6 === 'function' && _cdbc_3148608122_6({
  "errors":[
    "Failed to process \"polygon-fill\" property: unable to compute ramp, invalid input syntax for type numeric: \"inf\""
  ],
  "errors_with_context":[
    {
      "type":"layer",
      "message":"Failed to process \"polygon-fill\"
      property: unable to compute ramp, invalid input syntax for type numeric: \"inf\"",
      "http_status":400,
      "subtype":"turbo-carto",
      "layer":{"id":"85f6a3b5-06fe-46c0-8460-d70759c25a42","index":1,"type":"mapnik","context":{"selector":"#layer","source":{"start":{"line":2,"column":3},"end":{"line":2,"column":92}}}}
    }
  ]
});
```

And this is how the error looks in the UI:

![screen shot 2017-08-02 at 15 49 26](https://user-images.githubusercontent.com/5909/28877467-bc1f0266-779c-11e7-845c-83158bad7f4b.png)

**Comment by jgoizueta** [comment]
### Acceptance second round [success :grin:]

The problems described above are fixed by this PR: #719 

With that out of the way the PR for this issue works flawlessly: we can see how Popups get the correct values for infinite and NaN numeric values:

![screen shot 2017-08-02 at 17 43 19](https://user-images.githubusercontent.com/5909/28882405-6f5e630e-77ab-11e7-92f8-3e3b577cb655.png)

![screen shot 2017-08-02 at 17 43 25](https://user-images.githubusercontent.com/5909/28882411-72d3164c-77ab-11e7-8536-3eae67d79e2e.png)
